CT.NH.YU.YUAG.PR2D12-mdn.xml

```diff
@@ -0,0 +1,191 @@
+                        </summary>
+
+                        <origin>
+                            <date evidence="#archaeology" notBefore="-0183" notAfter="-0182">183/2 BCE</date>
+                            <placeName ref="https://pleiades.stoa.org/places/980015057">Syria, Dura-Europos, Palmyrene Gate</placeName>
+                            <p>south wall of the passage, flanking the innermost entrance to the city</p>
+                        </origin>
+                        <provenance type="found">
+                            <p>Excavated and recorded by <orgName>Yale-French Excavations at Dura-Europos</orgName> in 1928-1929.</p>
+                        </provenance>
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Dura-Europos"/>
+                        <biblScope unit="page">00</biblScope>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2022-08-11" who="James Flynn">Initial Encoding</change>
+            <change when="2022-11-23" who="Ilaria Bucci">Edited; general revision</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+    
+    <facsimile>
+        <surface>
+            <desc>Photograph (1929) of the tracings D9-D12, D123-D124 (YUAG)</desc>
+            <graphic url="https://library.artstor.org/#/asset/SS7730479_7730479_10869614"/>
+        </surface>
+        <surface>
+            <desc/>
+            <graphic url="https://commons.wikimedia.org/wiki/File:Palmyrene_Gate_D9-12,_123-4.jpg"/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/><persName><name key="Ἀρτεμίδωρος Artemidorus">Ἀρτεμίδωρος</name></persName><choice><unclear reason="faded">λρ'</unclear><unclear reason="faded">Αρ</unclear></choice>
```

Comment:
I think something is off with the use of choice here; if it's a correction, it <choice><sic>whats there</sic><corr>what you think it should be</corr></choice>. Otherwise though I think something is missing.

Reply:
I.e. it needs to be clear what the choice is between, not just that a choice is being made. There's good discussion in the commentary, but I think we'd like to settle on one or the other in the edition.

Reply:
Thank you for pointing this out! I'll settle for the one that looks more plausible based on the drawing (I don't have a photograph of this text) and I'll leave the rest for the commentary.